### DISCUSSION TITLE
Android TV support

### DISCUSSION THREAD

**Discussion by sjolsthoorn** [ionic-team/capacitor#3225] [Ideas]
As a big Android TV user, I started to make my own apps for it. I already make apps for Android and iOS using Ionic and thought I would give it a shot with Android TV, now you can simply make a big screen app, add some button mapping and sideload the APK to your Android TV and you got a app running but Google wants you to use the Leanback Launcher and its functionality in order to actually launch and use the app.

I already scouted for community plugins but there just about none (none with proper descriptions anyway), Android TV is not dead and used by many, development and app support for the OS is just very poor and Capacitor support would help making the platform easier to develop for, my own apps already run smooth on the TV, it just needs some extra configuration to make it run but official support from the Ionic Team would get a long way

**Comment by imhoffd** [comment]
This isn't something the Ionic Team will create, but please feel free to [create a plugin proposal](https://github.com/capacitor-community/proposals/) to the Capacitor Community to add Android TV support.

**Comment by baukef** [comment]
Hi Sjors, sounds like a very interesting approach. Do you have further insights that you can share?

**Reply by josefnorlin** [reply]
I too am interested in this. @sjolsthoorn, is that something you could share?

**Comment by folsze** [comment]
is this answer true?


> Ionic app builded for Android applications supports Android TV.
> 
> Also you can create separated Tizen project with web (html, css, js) architecture. Use iframe in that tizen project and you can use your app in Tizen platform. But for good navigation you need use CAPH lib.


https://stackoverflow.com/a/71603535/20009330

**Comment by kivileo9-sudo** [comment]
Writing

Totally agree! Android TV has a lot of potential, but better Capacitor support would make development much easier. We’ve seen this with [Xuper TV](https://thexuper.com/) too—official Ionic support could really help developers bring more apps to the TV platform.
